packages/connect-query/package.json

```diff
@@ -52,7 +52,7 @@
   "peerDependencies": {
     "@bufbuild/protobuf": "^1.3.0",
     "@connectrpc/connect": "^1.0.0",
-    "@tanstack/react-query": "^4.20.4",
+    "@tanstack/react-query": "4.x",
```

Comment:
This is the only change that isn't just a pure dependency update. Our API means that we only really care about the core API of react-query so we can be pretty permissive and only care about major vesion.